pkg/service/update/update.go

```diff
@@ -0,0 +1,141 @@
+package updateBinary
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/charmbracelet/glamour"
+	"go.keploy.io/server/utils"
+	"go.uber.org/zap"
+)
+
+// GitHubRelease holds information about the GitHub release.
+type GitHubRelease struct {
```

Comment:
Why do you have two structs for GitHubRelease..?

Reply:
Refactored